### DISCUSSION TITLE
Ubuntu shows in Windows "Unused Apps"

### DISCUSSION THREAD

**Discussion by davidgilbertson** [microsoft/WSL#9823] [General]
### Windows Version

Microsoft Windows [Version 10.0.22621.1265]

### WSL Version

1.1.3.0 

### Are you using WSL 1 or WSL 2?

- [X] WSL 2
- [ ] WSL 1

### Kernel Version

5.15.90.1

### Distro Version

Ubuntu 20.04

### Repro Steps

In Windows settings, I see this:

![image](https://user-images.githubusercontent.com/4443482/224830906-329f4950-53fd-4972-99d7-9bcd7fc2b6d2.png)

What is the "Ubuntu on Windows" entry? I use WSL every day, so it should not show up as "unused". If I were to remove this, would WSL stop working?

A side note, I think the "Windows subsystem for Linux" only showed up this morning after I ran `wsl --update` and clearly has a wrong date, but I'm less concerned about that because it's not 242GB.

### Expected Behavior

If this entry _does_ refer to my WSL install, it should not show as an unused app.

If it refers to something else, there should be some way for me to know that removing this will not break my WSL install.

### Actual Behavior

As above

**Comment by davidgilbertson** [comment]
I'm not sure, but I've just looked back in settings and both apps have gone from the 'unused' list. The date for "Windows Subsystem for Linux" is now correct, but the date for "Ubuntu on Windows" has changed to "12/3/2022"
![image](https://user-images.githubusercontent.com/4443482/226153663-855477c8-50b8-491a-9467-81e018ed3af8.png)

I don't know if these are issues with WSL or Windows itself, but I'd love it if someone could tell me whether or not uninstalling "Ubuntu on Windows" will break my current WSL installation, or if it's wasting a quarter-terabyte of space unnecessarily.

**Comment by elsaco** [comment]
That looks like an old Ubuntu WSL distro. See https://apps.microsoft.com/store/detail/ubuntu-on-windows/9NBLGGH4MSV6?hl=en-us&gl=us&rtc=1 for details. If there's no data you want to save from that distro then its safe to remove it. It won't affect your WSL installation.

**Comment by Greg0s** [comment]
Hello, I had the same problem and tried to uninstall it. I thought it was an older version but it turned out to be my current Ubuntu WSL installation :( I don't know why it was marked as "unused" as I use it almost every day.
